src/main/wiki/JVMMemory/GarbageCollection.md

```diff
@@ -0,0 +1,68 @@
+## Garbage Collection
```

Comment:
This file should be in "docs/jvm/garbage_collection.md".

Reply:
so the default format is snakecase?